src/utilities/backendValidation.ts

```diff
@@ -0,0 +1,70 @@
+import { isArray, isPlainObject, extend } from 'lodash';
+
+import { varToLabel } from '@mighty-justice/utils';
+
+// Takes an API response and converts it to a string to string map
```

Comment:
Kinda. I guess this is much more about taking what Django Rest Framework response with and turning it into a much simpler object. I don't think it could be applied anywhere else, especially since we throw away some data ( if an array is returned with multiple values ).